psi-sample/common/src/main/java/com/r3/conclave/sample/common/InputDataSerializer.java

```diff
@@ -0,0 +1,61 @@
+package com.r3.conclave.sample.common;
```

Comment:
I haven't used kryo before so I haven't looked at this file in detail.

Reply:
I have written custom serializers and de serializers which is used by Kryo to serialize and deserialize the data from client to the enclave and vice versa.